src/languages/es.js

```diff
@@ -677,8 +677,8 @@ export default {
             dob: 'Ingresa una fecha de nacimiento válida',
             age: 'Debe ser mayor de 18 años',
             ssnLast4: 'Ingresa los últimos 4 dígitos del número de seguro social',
-            firstName: 'Ingresa un nombre válido',
-            lastName: 'Ingresa un apellido válido',
+            firstName: 'Ingresa un nombre',
```

Comment:
To make it consistent with the lastName one.

```suggestion
            firstName: 'Ingresa el nombre',
```